core/object/script_language.h

```diff
@@ -373,6 +373,8 @@ class ScriptLanguage : public Object {
 	virtual void auto_indent_code(String &p_code, int p_from_line, int p_to_line) const = 0;
 	virtual void add_global_constant(const StringName &p_variable, const Variant &p_value) = 0;
 	virtual void add_named_global_constant(const StringName &p_name, const Variant &p_value) {}
+	virtual bool has_any_global_constant(const StringName &p_name) { return false; };
+	virtual Variant get_any_global_constant(const StringName &p_name) { return Variant(); };
```

Comment:
if this is a virtual method on ScriptLanguage, then csharp and other languages should also implement it,
it should probably be moved to a non-virtual method on GDScriptLanguage like `get_global_map` and `get_named_globals_map`